idl/partition_checksum.idl.hh

```diff
@@ -129,8 +134,10 @@ struct node_ops_cmd_request {
     std::list<gms::inet_address> leaving_nodes;
     // Map existing nodes to replacing nodes
     std::unordered_map<gms::inet_address, gms::inet_address> replace_nodes;
+    std::unordered_map<gms::inet_address, std::list<dht::token>> bootstrap_nodes;
 };
 
 struct node_ops_cmd_response {
     bool ok;
+    std::list<utils::UUID> pending_ops;
```

Comment:
Please document for which verbs this is expected to have a value.

These multiplexed verbs are hard to understand.

Reply:
Ok. Done.